index.html

```diff
@@ -4,83 +4,219 @@
+										<div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_9ab8c2f01a3f5aa677094fc1e_4c5085e68c" tabindex="-1" value=""></div>
+									</div>
+									<button type="submit" class="btn btn-first btn-email btn-fill">NOTIFY ME</button>
 								</div>
-								<button type="submit" class="btn btn-email btn-fill">Notify Me</button>
-
 							</form>
-							<div class="media">
-								<a href="#">
+
+							<div class="media mb-3">
+								<a href="https://www.linkedin.com/company/virtision-com">
 									<i class="fa fa-linkedin-square"></i>
 								</a>
 
 								<a href="#">
 									<i class="fa fa-facebook"></i>
 								</a>
 
-								<a href="#">
-									<i class="fa fa-envelope-o"></i>
+								<a href="mailto:team@virtision.com">
+									<i class="fa fa-envelope"></i>
 								</a>
 
 							</div>
 						</div>
 					</div>
 				</div>
+				<div class="col-lg-5 offset-lg-1 float-right mb-3">
+					<video playsinline="" muted="" class="rounded shadow-lg video" src="images/demo.mp4" loop autoplay></video>
+				</div>
+
+			</div>
+		</div>
+	</div>
+	<div class="main">
+		<div class="container value">
+			<div class="row justify-content-center">
+				<div class="col-lg-8 center">
+					<h1 class="pop"> For any creatives at any level to create and share their imagination in XR</h1>
+					<!--Toy Faces created by Amrit Pal Singh (https://amritpaldesign.com/toy-faces)-->
+				</div>
+				<div class="col-lg-12">
+					<img style="width:inherit" class="rounded shadow-lg" src="./images/artist2.png">
+				</div>
+
+			</div>
+		</div>
+		<div class="container value">
+			<div class="row pt-5">
+				<div class="col-lg-6">
+					<img class="demo" src="./images/creation.png" width="400">
+				</div>
+				<div class="col-lg-6">
+					<h1 class="pop">Simply create your XR story with just a few taps</h1>
+					<h3 class="motto">experience a unified and simplified creation process to unblock your creativity</h3>
+				</div>
+
+			</div>
+		</div>
+		<div class="container value">
+			<div class="row">
+				<div class="col-lg-6 order-md-2 order-sm-1">
+					<img class="demo" src="./images/sanbox.png" width="400">
+					<!--<a href='https://www.freepik.com/psd/woman'>Woman psd created by freepik - www.freepik.com</a>-->
+				</div>
+				<div class="col-lg-6  order-md-1 order-sm-2">
+					<h1 class="pop">The XR sandbox for your creativity</h1>
+					<h3 class="motto">powered by the community-built 3D assets, music, animation and effects library</h3>
+
+				</div>
+
+			</div>
+		</div>
+		<div class="container value">
+			<div class="row">
 				<div class="col-lg-6">
-					<video playsinline="" muted="" class="img-rounded video" src="images/demo.mp4" style="width:inherit" loop autoplay=""></video>
+					<img class="demo" src="./images/alldevice2.png" width="400">
 				</div>
+				<div class="col-lg-6">
+					<h1 class="pop">Share and enjoy many XR creations together in any device</h1>
```

Comment:
-> Share and enjoy many XR creations together **on** any device.

Kinda feel like this one could use a little overhaul.

"Share your XR creations with any device" 
"Share and enjoy a universe of XR creations with any device"
"Share and enjoy a world of XR creations with any device"

Reply:
I choose this ""Share and enjoy a universe of XR creations with any device"